metrics/core/src/main/java/tech/pegasys/pantheon/metrics/MetricsSystem.java

```diff
@@ -56,6 +56,6 @@ default void createLongGauge(
   Stream<Observation> streamObservations(MetricCategory category);
 
   default Stream<Observation> streamObservations() {
-    return Stream.of(MetricCategory.values()).flatMap(this::streamObservations);
+    return Stream.of(PantheonMetricCategory.values()).flatMap(this::streamObservations);
```

Comment:
This still ties us to `PantheonMetricCategory` but will be cleaned up in a follow-up PR. Trying to focus just on the rename in this one.